tests/QARegressionTests/test_Issues/test_CoreIssue576.py

```diff
@@ -0,0 +1,124 @@
+import subprocess
+import os
+import openpyxl
+import pytest
+from conftest import get_python_executable
+
+
+@pytest.mark.regression
+def test_negative_dataset():
+    command = (
+        f"{get_python_executable()} -m core test -s sdtmig -v 3.4 -r "
+        + os.path.join("tests", "resources", "CoreIssue576", "rule.json")
+        + " -dp "
+        + os.path.join("tests", "resources", "CoreIssue576", "Negative_Dataset.json")
```

Comment:
I'm not sure the 576 positive and negative test data is appropriate. It seems we should be testing that --SUPP datasets get included  and non-supps don't get included in the scope.

Reply:
I think right now we have only these datasets from the resources for unit tests. Maybe we can request a dataset that is according to what you have mentioned in your comment.

Reply:
I have updated the test  and datasets both. The positive dataset have only supp datasets. The test checks if the rule was successfully executed or not along with some other information. The negative dataset has no supp dataset and it checks if in such case the rule is skipped.